javascript/javascript3/week3/ShoppingCart/ShopCartJS.js

```diff
@@ -0,0 +1,87 @@
+const productContainer = document.getElementById("products-container");
+
+class Product {
+  constructor(name, price) {
+    this.name = name;
+    this.price = price;
+  }
+}
+
+class ShoppingCart {
+  constructor() {
+    this.products = [];
+  }
+
+  addProduct(product) {
+    this.products.push(product);
+    this.currency = product.currency;
+  }
+
+  removeProduct(product) {
+    const index = this.products.indexOf(product);
+    if (index > -1) {
+      this.products.splice(index, 1);
+    }
+  }
+
+  searchProduct(productName) {
+    const searchedProduct = this.products.filter((product) =>
+      product.name.toLowerCase().includes(productName)
+    );
+    return console.log(searchedProduct);
+  }
+
+  renderProducts() {
+    this.products.forEach((product) => {
+      const productList = document.createElement("p");
+      productList.innerHTML = `<p>${product.name}: ${product.price}</p>`;
```

Comment:
`productList` is already a `<p>` element. We don't need to put another `<p>` element inside it. We can just say 
```
productList.innerHTML = `${product.name}: ${product.price}`;
```